src/Compression/CompressionFactory.h

```diff
@@ -74,6 +75,8 @@ class CompressionCodecFactory final : private boost::noncopyable
     /// Get codec by name with optional params. Example: LZ4, ZSTD(3)
     CompressionCodecPtr get(const String & compression_codec) const;
 
+    void getMutableColumns(MutableColumns & res_columns) const;
```

Comment:
Let's rename it and add a comment with a description. I'd recommend something like `fillCodecDescriptions`.

Reply:
Done